### ISSUE TITLE
When indicator menu is open, navigate using arrows and select with ENTER

### ISSUE THREAD

**Issue by Tudmotu** [Tudmotu/gnome-shell-extension-clipboard-indicator#9]
If the popup menu is shown, allow the user to navigate between different entries with the arrow keys, and select an option with the `ENTER` key.

This should include the non-clipboard entries, like 'clear history' and 'settings'.

**Comment by thdox** [comment]
TL;DR: +1

Long story: I just moved from Debian stable wheezy to Debian testing jessie. I was using parcellite as a clipboard manager. My usual workflow is: edit something, hit my keyboard shortcut to show history of clipboard, move down with arrow to select one, hit enter. And voilà, this is pasted.

I just installed: 
- gpaste, and associated gnome shell extension, customize the keyboard shortcut, use my keyboard shortcut, and everything is frozen, no keyboard anymore, no mouse click
- parcellite, my older favorite, keyboard shortcut is not working
- diodon, keyboard shortcut works intermitently, that is 1 out of 4
- this Clipboard indicator. I love the simplicity. It is just missing the "Enter" for chosing an older clip.
- well, absolutely no clipboard manager working on Debian testing jessie, the one that should be stable in a few months. Here you have a chance to be the only one clipboard manager working a such a distrib like Debian stable ! :-)

**Comment by frol** [comment]
Please, implement ENTER selection feature!

**Comment by frol** [comment]
Alright, I fixed it myself, though you will need one extra "arrow-UP" to initiate navigation... I couldn't find how to set focus on the first item automatically.
